data_steward/cdr_cleaner/cleaning_rules/remove_invalid_procedure_source_records.py

```diff
@@ -16,17 +16,22 @@
 
 # Project imports
 import constants.cdr_cleaner.clean_cdr as cdr_consts
+from sandbox import get_sandbox_dataset_id
 
-REMOVE_INVALID_PROCEDURE_SOURCE_CONCEPT_IDS_QUERY = """
-DELETE
+INTERMEDIARY_TABLE_NAME = 'procedure_occurrence_dc583'
+
+INVALID_PROCEDURE_SOURCE_CONCEPT_IDS_QUERY = """
+CREATE OR REPLACE TABLE
+`{project}.{sandbox_dataset}.{intermediary_table}`
```

Comment:
This line needs to be updated to 
```
`{project}.{sandbox_dataset}.{intermediary_table}` AS
```
It's invalid SQL without the `AS`

Reply:
updated!